packages/askui-nodejs/src/lib/download-binaries.ts

```diff
@@ -43,7 +45,10 @@ function getBinaryDownloadUrl(binaryVersion: string): string {
   return `${baseUrl}/${platform()}/${arch}/${binarySubPathsByPlatform[platform()][1]}`;
 }
 
-export function downloadServerBinaries(binaryVersion: string): Promise<void> {
+export function downloadServerBinaries(
+  binaryVersion: string,
+  proxyAgent?: { http: http.Agent, https: https.Agent },
```

Comment:
Are really both required? I would have expected something like
```suggestion
  proxyAgents: { http?: http.Agent, https?: https.Agent } = {},
```

That could also save you some writing inside line 61:
```
const downloadStream = got.stream(url, { agent: proxyAgents });
```
although I am not sure how got handles an empty `agent` object

Reply:
`http.Agent` is for **http** connections
`http.Agent` is for **https** connections

Our API is using only https connection, but I have the possibility to set booth agents. e.g. For local inference. 

Agents can be set in general independently.

Reply:
Exactly, this is why I commented that they may both be optional because you can set one without setting the other,

Reply:
As discusses: I want to set booth values instead of one, to minimize misconfiguration issues. --> We stick to proxyAgents are optional, but you have to set booth agents